.gitlab-ci.yml

```diff
@@ -105,6 +105,16 @@ ib-aws:
     - schutzbot/deploy.sh
     - ARTIFACTS=/tmp/ bash /usr/libexec/tests/osbuild-composer/aws.sh
 
+container_main:
+  extends: .deps
+  rules:
```

Comment:
Maybe, instead of using rules, we can use the "only" section.

```
only:
    - main
```

Reply:
Another thing is, when the code is merged, the CI will look for `:main` container, and not `:latest`. So in that case we can add the "rule" section and declare a CI variable to use the correct tag.

Reply:
Only and except are not being actively developed, and might get deprecated: https://docs.gitlab.com/ee/ci/yaml/index.html#only--except